rapthor/lib/screen.py

```diff
@@ -426,16 +441,20 @@ def make_matrix(self, t_start_index, t_stop_index, freq_ind, stat_ind, cellsize_
         ncpu : int, optional
             Number of CPUs to use (0 means all)
         """
+        global screen_ph, screen_amp_xx, screen_amp_yy, pp, x, y, var_dict
```

Comment:
I'm not a big fan of `global`. Is there no way to encapsulate these variables (e.g. in a class)?

Reply:
Yeah, me neither but it is used here as a way to avoid pickling of the arrays for multiprocessing.

Reply:
OK, that makes sense. I already thought so. Maybe good to put a comment in the code why you use globals.